Add public-images article on qemu/qcow (#261)
canonical/ubuntu-cloud-docs: public-images/public-images-reference/artifacts.rst

       One of the primary use cases of our published QCOW images is to use QEMU to create and manage virtual machines. Other providers, such as VirtualBox, can be used for this, or you can use a ``.img`` file to create a bootable USB drive or deploy in an embedded system.

>        See the How-To article :ref:`qcow-qemu` for instructions on using QCOW images with QEMU.

Reviewer: ```suggestion
       Refer to :ref:`qcow-qemu` for instructions on using QCOW images with QEMU.
```
